lib/nanoc/filters/relativize_paths.rb

```diff
@@ -64,11 +64,12 @@ def relativize_html_like(content, params)
       selectors  = params.fetch(:select, SELECTORS)
       namespaces = params.fetch(:namespaces, {})
       type       = params.fetch(:type)
+      save_with  = params.fetch(:save_with, nil)
```

Comment:
The name `save_with` is confusing out of context; what about `nokogiri_save_with`?